README.md

```diff
@@ -62,7 +62,6 @@ The best way to get a review is to ask for one. On GitHub, you can ask someone t
 1. Open a new browser tab, and work on the steps in your second tab while you read the instructions in this tab.
 1. We made a pull request for you from the `update-game` branch, so open that pull request.
 1. Under **Reviewers** on the right side of the screen, add yourself as a reviewer.
-1. Wait about 20 seconds then refresh this page for the next step.
```

Comment:
@dihydroJenoxide Updated the copy in https://github.com/skills/hello-github-actions to this to clarify the instruction is meant for the instructions page, not the pull request.

> Wait about 20 seconds for actions to run, then refresh this page (the one you're following instructions from) and an action will automatically close this step and open the next one.